junebug/stores.py

```diff
@@ -43,19 +50,37 @@ def load_property(self, id, key):
 class InboundMessageStore(BaseStore):
     '''Stores the entire inbound message, in order to later construct
     replies'''
-    def _get_key(self, channel_id, message_id):
-        return '%s:inbound_messages:%s' % (channel_id, message_id)
+    KEYSPACE = 'inbound_messages'
 
     def store_vumi_message(self, channel_id, message):
         '''Stores the given vumi message'''
-        key = self._get_key(channel_id, message.get('message_id'))
+        key = self.get_key(channel_id, message.get('message_id'))
         return self.store_property(key, 'message', message.to_json())
 
     @inlineCallbacks
     def load_vumi_message(self, channel_id, message_id):
         '''Retrieves the stored vumi message, given its unique id'''
-        key = self._get_key(channel_id, message_id)
+        key = self.get_key(channel_id, message_id)
         msg_json = yield self.load_property(key, 'message')
         if msg_json is None:
             returnValue(None)
         returnValue(TransportUserMessage.from_json(msg_json))
+
+
+class OutboundMessageStore(BaseStore):
+    '''Stores the event url, in order to look it up when deciding where events
+    should go'''
+    KEYSPACE = 'outbound_messages'
+
+    def store_vumi_message(self, channel_id, api_request, message):
+        '''Stores the event_url'''
```

Comment:
If the plan is for this method to only handle storing event urls, maybe the name should suggest that it stores event urls associated to a message?